tests/test_session.py

```diff
@@ -49,6 +48,52 @@ def session():
     return session
 
 
+def test_session_signature(monkeypatch):
```

Comment:
We should test that passing `refresh_token` as the only positional argument works.